src/main/java/org/testng/xml/Parser.java

```diff
@@ -23,9 +24,17 @@
 
   /** The URL to the deprecated TestNG DTD. */
   public static final String DEPRECATED_TESTNG_DTD_URL = "http://beust.com/testng/" + TESTNG_DTD;
+  private static final String HTTPS_DEPRECATED_TESTNG_DTD_URL = "https://beust.com/testng/" + TESTNG_DTD;
 
   /** The URL to the TestNG DTD. */
   public static final String TESTNG_DTD_URL = "http://testng.org/" + TESTNG_DTD;
+  public static final String HTTPS_TESTNG_DTD_URL = "https://testng.org/" + TESTNG_DTD;
+
+  private static final List<String> urls = Arrays.asList(
```

Comment:
Constants are in upper case. And be careful, your constant is currently modifiable.

Reply:
fixed